src/test/java/database/UserDatabaseGatewayTest.java

```diff
@@ -7,14 +7,14 @@
 import java.io.IOException;
 
 public class UserDatabaseGatewayTest {
-    @Test public void testLoadUsersFromEmptyFile() {
-        try {
-            UserDatabaseGateway userGateway = new UserDatabaseGateway("database/emptyUserFile.ser");
-        } catch (IOException e){
-            assert false;
-            throw new RuntimeException(e);
-        }
-    }
+//    @Test public void testLoadUsersFromEmptyFile() {
+//        try {
+//            UserDatabaseGateway userGateway = new UserDatabaseGateway("database/emptyUserFile.ser");
+//        } catch (IOException e){
+//            assert false;
```

Comment:
Please either uncomment or delete these changes on your final pull request--it does not look good or finished to have code like this.